### ISSUE TITLE
Marlin\src\lcd\u8g_fontutf8.c:11:22: fatal error: clib/u8g.h: No such file or directory

### ISSUE THREAD

**Issue by maido-39** [MarlinFirmware/Marlin#10423]
```
Marlin\src\lcd\u8g_fontutf8.c:11:22: fatal error: clib/u8g.h: No such file or directory
```
this error on bugfix-2.0.x , so 
it should changed 
`#include <clib/u8g.h>` to `#include <lib/u8g.h>`

in `Marlin\src\lcd\u8g_fontutf8.c and Marlin\src\lcd\u8g_fontutf8.h`

**Comment by Bergerac56** [comment]
As a temporary measure and as u8g.h is invoqued by multiple files, it seems easier to add a marlin\clib\ directory and drop u8g.h file in it

There is a pull request (#10419) concerning this issue. I suppose it will be fixed quickly

**Comment by waddyano** [comment]
This PR doesn't seem to fix my local LPC1768 platformio build. But what I did do is similar to the suggestion . I added a clib/u8g.h that just includes lib/u8g.h in a directory already on the include path

**Comment by p3p** [comment]
That PR only fixes Travis with a workaround not platformio builds, the root cause is a differing folder name between the default u8glib and our HAL version, https://github.com/MarlinFirmware/U8glib-HAL/pull/2 should fix that.

**Comment by Elenedeath** [comment]
go to file `Marlin\src\lcd\u8g_fontutf8.c` and change` #include <clib/u8g.h>` to `#include <lib/u8g.h>`
work fine for me to compile marlin 2 for lpc1768.
i compile with PlatformIO on VSCode on a windows 10 64Bits

**Comment by thinkyhead** [comment]
@yhfudev — Is this fixed with our latest patch as of today?

**Comment by p3p** [comment]
As long as PlatformIO pulls down a new copy of the library this is fixed, so clean then build.

**Comment by yhfudev** [comment]
The previous PR #10419 passed travis building test, with the path to
```
 "/home/travis/build/MarlinFirmware/Marlin/.piolibdeps/U8glib-HAL_ID1932/src/lib/u8g.h", 
while the merged version in the 2.0 branch didn't, it shows the path "/home/travis/build/MarlinFirmware/Marlin/.piolibdeps/U8glib-HAL_ID1932/src/clib/u8g.h".
when the PlatformIO changed the "lib" to "clib"?
```
EDIT: the Travis seems work now with the commit 74f4eb831295a274f8a6d58d7a30d517cc39de87.

EDIT: I got it, https://github.com/MarlinFirmware/U8glib-HAL/pull/2

**Comment by waddyano** [comment]
I now have a clean build without addition hacks - thanks everyone - but I had to somewhat beat platformio into submission to download the new version of the library by uninstalling it. It didn't seem to do it automatically even though I could manually download the zip and see it had changed. I tried platformio lib update and also just letting vscode do the build. Is there something else I should have done? I am new to platformio.
